cmd/kubeadm/app/cmd/completion_test.go

```diff
@@ -39,7 +39,7 @@ func TestNewCmdCompletion(t *testing.T) {
 	cmd := newCmdCompletion(&out, "")
 	parentCmd.AddCommand(cmd)
 	if err := parentCmd.Execute(); err != nil {
-		t.Errorf("Cannot exectute newCmdCompletion: %v", err)
+		t.Errorf("cannot exectute newCmdCompletion: %v", err)
```

Comment:
my comment here stands:
https://github.com/kubernetes/kubernetes/pull/97435/files#r547436562

the purpose of keeping error messages (from the "error" package) lowercase is so that they can be propagated and combined - e.g. `fmt.Printf("some message: %v, err)`

the "testing" package `t.Errorf`, `t.Fatalf` ... etc result in log messages and are final.
so according to https://github.com/kubernetes/kubernetes/issues/15863#issue-112202805 we should not be lowercasing these.

Reply:
unless a new `verify-` script is added in `/hack` that makes this consistent i don't think we should act on it.

Reply:
ok，I have got it, Thanks!